### ISSUE TITLE
Newlines in strings are replaced with actual newlines

### ISSUE THREAD

**Issue by KristofferC** [domluna/JLFmt.jl#6]
```
julia> str = """error("foo\\n\\nbar")""";

julia> print(str)
error("foo\n\nbar")
julia> print(format(str))
error("foo

bar")
```

Is this intended?

**Comment by domluna** [comment]
No it's not. Handling strings is the biggest pain point at the moment. Nonetheless, I'm surprised this doesn't work.

**Comment by domluna** [comment]
Fixed in #10
